js/projectsConstructor.js

```diff
@@ -1,40 +1,31 @@
 'use strict';
 
+var projects = [];
+
 function Project (projectsListObj) {
   this.title = projectsListObj.title;
   this.url = projectsListObj.url;
   this.githubURL = projectsListObj.githubURL;
   this.img = projectsListObj.img;
   this.skillsUsed = projectsListObj.skillsUsed;
-  Project.all.push(this);
 }
 
 Project.prototype.toHtml = function() {
-  var $newProject = $('.template').clone();
-  $newProject.removeClass('template');
-  $newProject.find('h3')
-             .text(this.title);
-  $newProject.find('a')
-             .eq(0).attr('href', this.url);
-  $newProject.find('a')
-             .eq(1).attr('href',this.githubURL);
-  $newProject.find('img')
-             .attr('src',this.img);
-  $newProject.find('p')
-             .eq(0).text(this.skillsUsed);
-  $('.anchorProjects').append($newProject);
+  var sourceHTML = $('#project-template').html();
+  var actualTemplate = Handlebars.compile(sourceHTML);
+  var newRawHTML = actualTemplate(this);
+  $('main').append(newRawHTML);
```

Comment:
You either should return `newRawHTML` here instead of appending or just invoke `project.toHtml()` by itself on line 25. Otherwise, the `append` on line 25 is not actually doing anything.